_posts/en/2023-03-08-podcast.md

```diff
@@ -0,0 +1,67 @@
+type: posts
+layout: post
+lang: en
+slug: 2023-03-08-podcast-announcement
+
+excerpt: >
+  Announcement of a new Optech podcast based on the weekly Optech Audio
+  Recap hosted on Twitter Spaces.
+
+---
+Since FIXME, Optech contributors Mike Schmidt and Mark "Murch" Erhardt
+have been joined on Twitter Spaces by members of the community and
+leading Bitcoin and LN protocol developers to discuss the events
+summarized in each week's Optech Newsletter.  Each of those discussions
+will now be available as episodes on [all major podcast platforms][podcast
+index].
+
+Each episode will also be available as a page on the Optech website with
+a summary of the episode content and a complete transcript.  We think
+this is major advance in Optech's mission to improve communication
+between open source projects and downstream users and businesses.
+
+Anyone who wants to keep track of Bitcoin and LN protocol development
+can subscribe to either the newsletter (if they like reading) or the
+podcast (if they prefer listening).  If they discover a topic through
+the newsletter that they want to know more about, they can find out what
+experienced Bitcoin contributors are saying about the topic by listening
+to the podcast episode or reading its transcript.  Or, if they first
+hear about an interesting topic through the podcast, they can use that
+week's newsletter to immediately find the original discussions on a
+mailing list or pull request.
+
+To make it easy to move between the newsletter and the podcast, both
+sources are cross-indexed to each other.  With one click at the end of a
+newsletter item, you'll be taken to corresponding item on a podcast
+episode.  From there, you can immediately jump to that part of the audio
+episode or the transcript.  From the summary at the top of each podcast
+page, each item links directly to the part of the original newsletter it
+describes.
+
+Years from now, users, developers, and researchers will be able to look
+up a subject in Optech's extensive [topic index][], choose a particular
+item of interest, review a brief summary of the event in the newsletter,
+and then listen or read contemporary discussion about the subject from
+users and experts---often including the experts who contributed the most
+to that subject and the users who felt the most strongly about it.
+
+Of course, the podcast audio and transcripts, like all Optech content,
+is available under an open source license.
+
+New podcast episodes will be released a few days after the conclusion of
+a Twitter Spaces to allow for professional post production of the audio.
+Transcripts will be published a few days after that.  In addition to
+publishing new episodes as soon as they've complete, Optech's audio and
+transcription teams will gradually be publishing old episodes.
+
+If you're interested in Bitcoin technology, we encourage you to
+subscribe to the podcast and consider joining us each week for a new
+recap on Twitter Spaces.
```

Comment:
Maybe add the time when Twitter Spaces are held here?

Reply:
I added 15:00 Thursday to the blog post in a comment.  I'll leave it up to @bitschmidty to decide whether or not to publish it with or without changes.